src/Data/provideLocalStorageDataClass.ts

```diff
@@ -142,6 +145,34 @@ export function provideLocalStorageDataClass(
   }
 }
 
+export function searchLocalStorageDataClasses(
+  search: string,
+  blackListEntities: string[] = [],
+): Array<{ _id: string; entity: string; title: string }> {
+  const results: Array<{ _id: string; entity: string; title: string }> = []
```

Comment:
* [ ] FYI For a reader with a functional programming background something like this could be more readable (actually this is what I translate the logic of the function to in my head when reading the code):

```tsx
  return entities.flatMap((entity) =>
    Object.entries(restoreRecord(recordKeyForClassName(entity)))
      .filter(([, item]) => Object.values(item).some(matchesSearchTerm))
      .map(([_id, item]) => ({
        _id,
        entity,
        title: ensureString(item.title) || ensureString(item.keyword),
      })),
  )
```